core/base/src/constants/rpc.ts

```diff
@@ -91,6 +92,7 @@ const rpcConfig = [[
     ["Snaxchain",       "https://testnet.snaxchain.io"],
     ["Unichain",        "https://sepolia.unichain.org"],
     ["Worldchain",      "https://worldchain-sepolia.g.alchemy.com/public"],
+    ["Ink",             "https://rpc-qnd-sepolia.inkonchain.com"],
```

Comment:
where did you get this from? https://chainlist.org/chain/763373 says it's `https://rpc-gel-sepolia.inkonchain.com` so double checking

Reply:
It came from their docs [here ](https://docs.inkonchain.com/quick-start/get-connected)and it's the one I used to deploy the contracts, so I think it's good.